packages/plugins/graphql/server/src/bootstrap.ts

```diff
@@ -115,14 +121,29 @@ export async function bootstrap({ strapi }: { strapi: Core.Strapi }) {
   const path: string = config('endpoint');
 
   const landingPage = determineLandingPage(strapi);
+  const pluginAddRootQueryArgs: ApolloServerPlugin<StrapiGraphQLContext> = {
```

Comment:
Can you add a comment for this? I'm worried all of this will end up getting forgotten about over time.

Reply:
yes sorry I thought I had one but must have deleted it on accident at some point